### PR TITLE
Issue 16 DegreeAv, IndegreeAv, OutDegreeAv

### PR DESCRIPTION
**Pull request by egolol**
Created DegreeAv, IndegreeAv, OutDegreeAv

### CODE REVIEW

File: braph/graph/DegreeAV.m [hunk 1]
```diff
@@ -0,0 +1,43 @@
+classdef DegreeAV < Measure
```
Comment (giovannivolpe) [inline braph/graph/DegreeAV.m]: classdef DegreeAV < Degree

File: braph/graph/DegreeAV.m [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    methods
+        function m = DegreeAV(g, varargin)
+            m = m@Measure(g, varargin{:});
```
Comment (giovannivolpe) [inline braph/graph/DegreeAV.m]: m = m@Degree(g, varargin{:});

File: braph/graph/DegreeAV.m [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    end
+    methods (Access=protected)
+       function calculate(m)
+            g = m.getGraph();          
+            % get degree measure 
+            % of the graph.
+            degree = g.getMeasure('Degree');
+            
+            av_degree = mean(degree.value,'all');
+            m.setValue(av_degree);
```
Comment (giovannivolpe) [inline braph/graph/DegreeAV.m]: degree = calculate@Degree(g);
degree_av = mean(degree);
m.setValue(degree_av);

File: braph/graph/InDegreeAV.m [hunk 4]
```diff
@@ -0,0 +1,44 @@
+classdef InDegreeAV < Measure
```
Comment (giovannivolpe) [inline braph/graph/InDegreeAV.m]: change as above

File: braph/graph/OutDegreeAV.m [hunk 5]
```diff
@@ -0,0 +1,44 @@
+classdef OutDegreeAV < Measure
```
Comment (giovannivolpe) [inline braph/graph/OutDegreeAV.m]: change as above

**Comment by egolol** [comment]
Changed to new way

File: braph/graph/OutDegreeAV.m (hunk 5 above)
Comment (egolol) [reply braph/graph/OutDegreeAV.m]: changed

File: braph/graph/InDegreeAV.m (hunk 4 above)
Comment (egolol) [reply braph/graph/InDegreeAV.m]: changed

File: braph/graph/DegreeAV.m (hunk 2 above)
Comment (egolol) [reply braph/graph/DegreeAV.m]: changed

**Pull request merged**